<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="StartADCCollection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>560</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OnceCollection">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>500</y>
      <width>80</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>551</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QPushButton" name="StopADCCollection">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>560</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭采集</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connnectbutton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>连接设备</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateConfig">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>下发配置</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>391</width>
      <height>481</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>采集卡控制</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道3量程</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道4量程</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>采样率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sampleRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>5</number>
     </property>
     <item>
      <property name="text">
       <string>125M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>50K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1K</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道1量程</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道2量程</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CH1_AC_DC">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>触发方式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="triggerType">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>滚动触发</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>上升沿触发</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>下降沿触发</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>双边触发</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>触发通道</string>
     </property>
    </widget>
    <widget class="QComboBox" name="triggerCH">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>CH1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CH4</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>触发电平</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>21</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>mv</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="triggerlevel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>采样点数</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sampleNum">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>4920</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>超时时间</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timeout">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="enable_CH1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>使能通道1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="enable_CH3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>使能通道3</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="enable_CH2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>使能通道2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="enable_CH4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>使能通道4</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>350</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>通道3</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CH2_AC_DC">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>350</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DC</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CH3_AC_DC">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>320</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DC</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CH4_AC_DC">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>350</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DC</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>文件保存路径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="saveFilePath">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>C:/Users/Administrator.DESKTOP-04T6SA5/Desktop/DAQ217_qt/KK/TEST</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectPath">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>440</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>文件保存模式：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="saveMode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>440</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>连续存储</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>单个文件限制：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="maxFileLineNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>410</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>100000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>410</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>行</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CH1_MaxV">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>-/+1V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-/+10V</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CH3_MaxV">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>-/+1V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-/+10V</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CH2_MaxV">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>-/+1V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-/+10V</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="CH4_MaxV">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>-/+1V</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>-/+10V</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Inter','-apple-system','BlinkMacSystemFont','Segoe UI','SF Pro SC','SF Pro Display','SF Pro Icons','PingFang SC','Hiragino Sans GB','Microsoft YaHei','Helvetica Neue','Helvetica','Arial','sans-serif'; font-size:16px; font-weight:600; color:#000000;&quot;&gt;数据长度&lt;/span&gt;&lt;span style=&quot; font-family:'Inter','-apple-system','BlinkMacSystemFont','Segoe UI','SF Pro SC','SF Pro Display','SF Pro Icons','PingFang SC','Hiragino Sans GB','Microsoft YaHei','Helvetica Neue','Helvetica','Arial','sans-serif'; font-size:16px; color:#000000;&quot;&gt;：从触发时刻开始，采集卡会持续采集数据，直到达到预设的采集点数，这个预设的采集点数就是触发数据长度。例如，如果触发数据长度设置为 1000 个点，那么采集卡会在触发后采集 1000 个数据点，然后停止采集（除非设置为连续采集模式）。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;</string>
     </property>
     <property name="text">
      <string>触发数据长度</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="triggerLength">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>50000</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>触发模式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="triggerMode">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>自动</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>普通</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="countTime">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>单通道累加次数</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
